src/reducers/SensorDetailReducer.js

```diff
@@ -32,12 +32,18 @@ export const SensorDetailReducer = (state = initialState(), action) => {
       const { payload } = action;
       const { name } = payload;
 
+      const isValid = name.length < 50;
+      if (!isValid) return state;
+
       return { ...state, name };
     }
     case SENSOR_DETAIL_ACTIONS.UPDATE_CODE: {
       const { payload } = action;
       const { code } = payload;
 
+      const isValid = code.length < 50;
```

Comment:
Should this be 10 as well? (unless we're intentionally being more restrictive when creating?)

Reply:
Whoopsies!


4c52af3

🙈